docs/UserGuide.md

```diff
@@ -268,17 +411,25 @@ _Details coming soon ..._
+| **Comment**             | `comment INDEX COMMENT` <br> e.g., `comment 3 Managed to solve every round 3 interview questions.`                                                  |
+| **Delete**              | `delete INDEX` <br> e.g., `delete 3`                                                                                                                |
+| **Delete job**          | `delete_job INDEX` <br> e.g., `delete_job 3`                                                                                                        |
+| **Delete application**  | `delete_app INDEX` <br> e.g., `delete_app 3`                                                                                                        |
+| **Edit**                | `edit INDEX [n/NAME] [e/EMAIL] [c/COUNTRY] [p/PHONE] [t/TAG]…​` <br> e.g.,`edit 24 n/Johnny Doe e/johnnydoe@gmail.com c/SG`                         |
+| **Edit job**            | `edit_job INDEX [ti/TITLE] [d/DESCRIPTION] [v/VACANCY]` <br> e.g., `edit_job 1 ti/Quantitative Trader d/Must have strong statistics background v/3` |
+| **Get**                 | `get INDEX` <br> e.g., `get 24`                                                                                                                     |
+| **Search**              | `search [n/NAME] [e/EMAIL] [c/COUNTRY] [m/COMMENT] [p/PHONE] [t/TAG]` <br> e.g., `search n/John c/SG`                                               |
+| **Search jobs**         | `search_job [ti/TITLE] [d/DESCRIPTION] [v/VACANCY]` <br> e.g., `search_job ti/Engineer d/C++`                                                       |
+| **Search applications** | `search_app [e/EMAIL] [ti/TITLE] [s/STATUS]` <br> e.g., `search_app e/alexyeoh@example.com s/PRESCREEN`                                             |
+| **Tag**                 | `tag INDEX [t/TAG]…` <br> e.g., `tag 8 t/Exceptional work t/IMO gold t/Male`                                                                        |
+| **Status**              | `status INDEX INTERVIEW_STATUS` <br> e.g., `status 24 IN_PROGRESS`                                                                                  |
+| **List**                | `list`                                                                                                                                              |
+| **List jobs**           | `list_job`                                                                                                                                          |
+| **List applications**   | `list_applications`                                                                                                                                 |
+| **Help**                | `help`                                                                                                                                              |
```

Comment:
seems like forgot slots_left, delete_tag commands?

Reply:
Thanks for the catch, fixed